conda_build/windows.py

```diff
@@ -66,6 +66,16 @@ def msvc_env_cmd(override=None):
         msvc_env_lines.append('set DISTUTILS_USE_SDK=1')
         msvc_env_lines.append('set MSSdk=1')
 
+    if version == '10.0-SDK':
+        # Special override to use the MSVC 2010 compiler (10.0) from Microsoft SDK 7.1
+        setenv = os.path.join(os.environ['ProgramFiles'], 'Microsoft SDKs', 'Windows', 'v7.1',
```

Comment:
2010 compiler is not available in other (there is/will be another?) windows sdk?

Don't is better to get this path from registry? Maybe it is installed in a non standard place?
Key: HKEY_LOCAL_MACHINE
\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v7.1\InstallationFolder